script/upgrades/MU03/MU03Checks.sol

```diff
@@ -744,8 +710,8 @@ contract MU03Checks is Script, Test {
   ) internal view {
     if (currentThreshold != expectedThreshold) {
       if (isValueDeltaBreaker) {
-        console2.log("ValueDeltaBreaker rate change threshold not set correctly for rate feed %s", rateFeedID);
-        revert("ValueDeltaBreaker rate change threshold not set correctly for all rate feeds");
+        console2.log("ValueDeltaBreaker rate change threshold not set correctly for cUSD/USDC %s", rateFeedID);
+        revert("ValueDeltaBreaker rate change threshold not set correctly for all cUSD/USDC");
```

Comment:
ditto, and I think here we probably don't want `cUSD/USDC` written at all right? this is a function called with a rateFeedID that can be any one.

Reply:
since we use valueDeltaBreaker only for USDC/USD rate feed I decided to make it specific in mu03. should I change it?